remove ggplot2Animint dependency

vivekktiwari opened the pull request:
@tdhock test suite is passing. 

sorry for extra PRs had to move all codes to `merge` branch

tdhock commented:
nice to hear that all the tests are passing. I will review some time in the
next few days before merging.

On Fri, Jun 15, 2018 at 2:25 AM, Vivek Kumar <notifications@github.com>
wrote:

> @tdhock <https://github.com/tdhock> test suite is passing.
>
> sorry for extra PRs had to move all codes to merge branch
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/tdhock/animint2/pull/23
> Commit Summary
>
>    - delete this file
>    - file deleted
>    - renaming animint files with a_*
>    - adding ggplot2Animint files -1
>    - adding ggplot2Animint files -2
>    - add collate to animint2 and ggplot2Animint files
>    - removing ggplot2Animint from Description.
>    - change in syntax of imported files from ggplot2Animint
>    - add import and suggest from ggplot2Animint
>    - remove ggplot2Animint from NAMESPACE
>    - export NAMESPACE of ggplot2Animint and animint files changes to z_*
>    to avoid error from collate
>    - data files from ggplot2Animint
>    - raw data files and .Rd files from ggplot2Animint and animint2
>    - rename similar function with g_* and minor edits
>    - new GITHUB_PAT
>    - new GITHUB_PAT
>    - Update .travis.yml
>    - Merge branch 'merge' into cran
>    - Merge pull request #18 from vivekktiwari/cran
>    - changing few function names to a_*
>    - Merge branch 'cran' of https://github.com/vivekktiwari/animint2 into
>    cran
>    - Merge pull request #21 from vivekktiwari/cran
>    - update DESCRIPTION
>    - Merge pull request #22 from vivekktiwari/cran
>    - Merge branch 'merge' of https://github.com/tdhock/animint2 into merge
>    - remove ggplot2Animint from example
>
> File Changes
>
>    - *M* .gitignore
>    <https://github.com/tdhock/animint2/pull/23/files#diff-0> (1)
>    - *M* .travis.yml
>    <https://github.com/tdhock/animint2/pull/23/files#diff-1> (4)
>    - *M* DESCRIPTION
>    <https://github.com/tdhock/animint2/pull/23/files#diff-2> (208)
>    - *M* NAMESPACE
>    <https://github.com/tdhock/animint2/pull/23/files#diff-3> (496)
>    - *A* R/aaa-.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-4> (13)
>    - *A* R/aes-calculated.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-5> (50)
>    - *A* R/aes-colour-fill-alpha.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-6> (56)
>    - *A* R/aes-group-order.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-7> (80)
>    - *A* R/aes-linetype-size-shape.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-8> (64)
>    - *A* R/aes-position.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-9> (39)
>    - *A* R/aes.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-10> (235)
>    - *A* R/annotation-custom.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-11> (97)
>    - *A* R/annotation-logticks.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-12> (244)
>    - *A* R/annotation-map.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-13> (72)
>    - *A* R/annotation-raster.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-14> (90)
>    - *A* R/annotation.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-15> (67)
>    - *A* R/autoplot.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-16> (21)
>    - *A* R/bench.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-17> (24)
>    - *A* R/bin.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-18> (171)
>    - *A* R/coord-.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-19> (81)
>    - *A* R/coord-cartesian-.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-20> (103)
>    - *A* R/coord-fixed.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-21> (47)
>    - *A* R/coord-flip.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-22> (70)
>    - *A* R/coord-map.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-23> (249)
>    - *A* R/coord-munch.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-24> (197)
>    - *A* R/coord-polar.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-25> (304)
>    - *A* R/coord-quickmap.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-26> (36)
>    - *A* R/coord-transform.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-27> (174)
>    - *A* R/data.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-28> (194)
>    - *A* R/facet-.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-29> (62)
>    - *A* R/facet-grid-.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-30> (415)
>    - *A* R/facet-labels.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-31> (575)
>    - *A* R/facet-layout.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-32> (175)
>    - *A* R/facet-locate.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-33> (84)
>    - *A* R/facet-null.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-34> (76)
>    - *A* R/facet-viewports.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-35> (50)
>    - *A* R/facet-wrap.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-36> (463)
>    - *A* R/fortify-lm.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-37> (86)
>    - *A* R/fortify-map.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-38> (111)
>    - *A* R/fortify-multcomp.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-39> (84)
>    - *A* R/fortify-spatial.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-40> (109)
>    - *A* R/fortify.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-41> (27)
>    - *A* R/geom-.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-42> (178)
>    - *A* R/geom-abline.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-43> (129)
>    - *A* R/geom-bar.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-44> (131)
>    - *A* R/geom-bin2d.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-45> (43)
>    - *A* R/geom-blank.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-46> (38)
>    - *A* R/geom-boxplot.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-47> (228)
>    - *A* R/geom-contour.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-48> (72)
>    - *A* R/geom-count.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-49> (63)
>    - *A* R/geom-crossbar.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-50> (95)
>    - *A* R/geom-curve.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-51> (61)
>    - *A* R/geom-defaults.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-52> (41)
>    - *A* R/geom-density.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-53> (81)
>    - *A* R/geom-density2d.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-54> (77)
>    - *A* R/geom-dotplot.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-55> (285)
>    - *A* R/geom-errorbar.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-56> (58)
>    - *A* R/geom-errorbarh.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-57> (81)
>    - *A* R/geom-freqpoly.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-58> (25)
>    - *A* R/geom-hex.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-59> (99)
>    - *A* R/geom-histogram.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-60> (99)
>    - *A* R/geom-hline.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-61> (54)
>    - *A* R/geom-jitter.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-62> (62)
>    - *A* R/geom-label.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-63> (147)
>    - *A* R/geom-linerange.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-64> (88)
>    - *A* R/geom-map.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-65> (123)
>    - *A* R/geom-path.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-66> (314)
>    - *A* R/geom-point.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-67> (144)
>    - *A* R/geom-pointrange.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-68> (49)
>    - *A* R/geom-polygon.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-69> (114)
>    - *A* R/geom-quantile.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-70> (69)
>    - *A* R/geom-raster.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-71> (93)
>    - *A* R/geom-rect.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-72> (81)
>    - *A* R/geom-ribbon.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-73> (143)
>    - *A* R/geom-rug.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-74> (95)
>    - *A* R/geom-segment.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-75> (117)
>    - *A* R/geom-smooth.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-76> (135)
>    - *A* R/geom-spoke.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-77> (65)
>    - *A* R/geom-text.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-78> (218)
>    - *A* R/geom-tile.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-79> (104)
>    - *A* R/geom-violin.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-80> (177)
>    - *A* R/geom-vline.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-81> (54)
>    - *A* R/ggplot2.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-82> (4)
>    - *A* R/ggproto.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-83> (285)
>    - *A* R/grob-absolute.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-84> (47)
>    - *A* R/grob-dotstack.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-85> (46)
>    - *A* R/grob-null.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-86> (24)
>    - *A* R/grouping.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-87> (39)
>    - *A* R/guide-colorbar.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-88> (440)
>    - *A* R/guide-legend.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-89> (519)
>    - *A* R/guides-.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-90> (293)
>    - *A* R/guides-axis.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-91> (115)
>    - *A* R/guides-grid.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-92> (30)
>    - *A* R/hexbin.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-93> (38)
>    - *A* R/labels.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-94> (94)
>    - *A* R/layer.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-95> (320)
>    - *A* R/legend-draw.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-96> (201)
>    - *A* R/limits.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-97> (132)
>    - *A* R/margins.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-98> (168)
>    - *A* R/panel.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-99> (185)
>    - *A* R/plot-build.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-100> (299)
>    - *A* R/plot-construction.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-101> (123)
>    - *A* R/plot-last.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-102> (19)
>    - *A* R/plot.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-103> (166)
>    - *A* R/position-.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-104> (92)
>    - *A* R/position-collide.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-105> (110)
>    - *A* R/position-dodge.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-106> (60)
>    - *A* R/position-fill.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-107> (24)
>    - *A* R/position-identity.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-108> (17)
>    - *A* R/position-jitter.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-109> (58)
>    - *A* R/position-jitterdodge.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-110> (68)
>    - *A* R/position-nudge.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-111> (46)
>    - *A* R/position-stack.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-112> (74)
>    - *A* R/quick-plot.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-113> (158)
>    - *A* R/range.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-114> (33)
>    - *A* R/save.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-115> (135)
>    - *A* R/scale-.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-116> (631)
>    - *A* R/scale-alpha.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-117> (35)
>    - *A* R/scale-brewer.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-118> (101)
>    - *A* R/scale-continuous.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-119> (168)
>    - *A* R/scale-date.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-120> (162)
>    - *A* R/scale-discrete-.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-121> (141)
>    - *A* R/scale-gradient.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-122> (113)
>    - *A* R/scale-grey.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-123> (39)
>    - *A* R/scale-hue.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-124> (59)
>    - *A* R/scale-identity.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-125> (154)
>    - *A* R/scale-linetype.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-126> (29)
>    - *A* R/scale-manual.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-127> (86)
>    - *A* R/scale-shape.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-128> (41)
>    - *A* R/scale-size.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-129> (94)
>    - *A* R/scale-type.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-130> (66)
>    - *A* R/scales-.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-131> (128)
>    - *A* R/stat-.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-132> (138)
>    - *A* R/stat-bin.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-133> (140)
>    - *A* R/stat-bin2d.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-134> (145)
>    - *A* R/stat-bindot.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-135> (172)
>    - *A* R/stat-binhex.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-136> (52)
>    - *A* R/stat-boxplot.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-137> (98)
>    - *A* R/stat-contour.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-138> (113)
>    - *A* R/stat-count.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-139> (71)
>    - *A* R/stat-density-2d.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-140> (74)
>    - *A* R/stat-density.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-141> (102)
>    - *A* R/stat-ecdf.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-142> (77)
>    - *A* R/stat-ellipse.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-143> (119)
>    - *A* R/stat-function.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-144> (103)
>    - *A* R/stat-identity.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-145> (39)
>    - *A* R/stat-qq.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-146> (91)
>    - *A* R/stat-quantile.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-147> (94)
>    - *A* R/stat-smooth-methods.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-148> (71)
>    - *A* R/stat-smooth.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-149> (132)
>    - *A* R/stat-sum.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-150> (50)
>    - *A* R/stat-summary-2d.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-151> (135)
>    - *A* R/stat-summary-bin.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-152> (89)
>    - *A* R/stat-summary-hex.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-153> (52)
>    - *A* R/stat-summary.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-154> (225)
>    - *A* R/stat-unique.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-155> (39)
>    - *A* R/stat-ydensity.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-156> (109)
>    - *A* R/summary.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-157> (42)
>    - *A* R/theme-defaults.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-158> (309)
>    - *A* R/theme-elements.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-159> (0)
>    - *A* R/theme.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-160> (0)
>    - *A* R/translate-qplot-ggplot.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-161> (0)
>    - *A* R/translate-qplot-lattice.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-162> (0)
>    - *A* R/utilities-break.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-163> (0)
>    - *A* R/utilities-grid.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-164> (0)
>    - *A* R/utilities-help.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-165> (0)
>    - *A* R/utilities-matrix.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-166> (0)
>    - *A* R/utilities-resolution.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-167> (0)
>    - *A* R/utilities-table.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-168> (0)
>    - *A* R/utilities.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-169> (0)
>    - *A* R/uu_zxx.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-170> (0)
>    - *A* R/uu_zzz.r
>    <https://github.com/tdhock/animint2/pull/23/files#diff-171> (0)
>    - *R* R/z_animint.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-172> (0)
>    - *R* R/z_animintHelpers.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-173> (0)
>    - *R* R/z_facets.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-174> (0)
>    - *R* R/z_geoms.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-175> (0)
>    - *R* R/z_gist.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-176> (0)
>    - *R* R/z_helperFunctions.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-177> (0)
>    - *R* R/z_knitr.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-178> (0)
>    - *R* R/z_print.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-179> (0)
>    - *R* R/z_scales.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-180> (0)
>    - *R* R/z_testHelpers.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-181> (0)
>    - *R* R/z_theme_animint.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-182> (0)
>    - *R* R/z_transformShape.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-183> (0)
>    - *A* data-raw/diamonds.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-184> (0)
>    - *A* data-raw/diamonds.csv
>    <https://github.com/tdhock/animint2/pull/23/files#diff-185> (0)
>    - *A* data-raw/economics.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-186> (0)
>    - *A* data-raw/economics.csv
>    <https://github.com/tdhock/animint2/pull/23/files#diff-187> (0)
>    - *A* data-raw/faithfuld.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-188> (0)
>    - *A* data-raw/luv_colours.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-189> (0)
>    - *A* data-raw/midwest.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-190> (0)
>    - *A* data-raw/midwest.csv
>    <https://github.com/tdhock/animint2/pull/23/files#diff-191> (0)
>    - *A* data-raw/mpg.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-192> (0)
>    - *A* data-raw/mpg.csv
>    <https://github.com/tdhock/animint2/pull/23/files#diff-193> (0)
>    - *A* data-raw/msleep.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-194> (0)
>    - *A* data-raw/msleep.csv
>    <https://github.com/tdhock/animint2/pull/23/files#diff-195> (0)
>    - *A* data-raw/presidential.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-196> (0)
>    - *A* data-raw/presidential.csv
>    <https://github.com/tdhock/animint2/pull/23/files#diff-197> (0)
>    - *A* data-raw/seals.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-198> (0)
>    - *A* data-raw/seals.csv
>    <https://github.com/tdhock/animint2/pull/23/files#diff-199> (0)
>    - *A* data-raw/tx-housing.R
>    <https://github.com/tdhock/animint2/pull/23/files#diff-200> (0)
>    - *A* data-raw/tx-housing.csv
>    <https://github.com/tdhock/animint2/pull/23/files#diff-201> (0)
>    - *A* data/diamonds.rda
>    <https://github.com/tdhock/animint2/pull/23/files#diff-202> (0)
>    - *A* data/economics.rda
>    <https://github.com/tdhock/animint2/pull/23/files#diff-203> (0)
>    - *A* data/economics_long.rda
>    <https://github.com/tdhock/animint2/pull/23/files#diff-204> (0)
>    - *A* data/faithfuld.rda
>    <https://github.com/tdhock/animint2/pull/23/files#diff-205> (0)
>    - *A* data/luv_colours.rda
>    <https://github.com/tdhock/animint2/pull/23/files#diff-206> (0)
>    - *A* data/midwest.rda
>    <https://github.com/tdhock/animint2/pull/23/files#diff-207> (0)
>    - *A* data/mpg.rda
>    <https://github.com/tdhock/animint2/pull/23/files#diff-208> (0)
>    - *A* data/msleep.rda
>    <https://github.com/tdhock/animint2/pull/23/files#diff-209> (0)
>    - *A* data/presidential.rda
>    <https://github.com/tdhock/animint2/pull/23/files#diff-210> (0)
>    - *A* data/seals.rda
>    <https://github.com/tdhock/animint2/pull/23/files#diff-211> (0)
>    - *A* data/txhousing.rda
>    <https://github.com/tdhock/animint2/pull/23/files#diff-212> (0)
>    - *M* inst/examples/test_knit_print.Rmd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-213> (0)
>    - *A* man/absoluteGrob.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-214> (0)
>    - *M* man/addSSandCSasAesthetics.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-215> (0)
>    - *M* man/addShowSelectedForLegend.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-216> (0)
>    - *A* man/add_theme.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-217> (0)
>    - *A* man/aes.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-218> (0)
>    - *A* man/aes_.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-219> (0)
>    - *A* man/aes_all.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-220> (0)
>    - *A* man/aes_auto.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-221> (0)
>    - *A* man/aes_colour_fill_alpha.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-222> (0)
>    - *A* man/aes_group_order.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-223> (0)
>    - *A* man/aes_linetype_size_shape.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-224> (0)
>    - *A* man/aes_position.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-225> (0)
>    - *A* man/animint2-ggproto.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-226> (0)
>    - *M* man/animint2dir.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-227> (0)
>    - *M* man/animint2gist.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-228> (0)
>    - *M* man/animintOutput.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-229> (0)
>    - *A* man/annotate.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-230> (0)
>    - *A* man/annotation_custom.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-231> (0)
>    - *A* man/annotation_logticks.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-232> (0)
>    - *A* man/annotation_map.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-233> (0)
>    - *A* man/annotation_raster.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-234> (0)
>    - *A* man/as.list.ggproto.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-235> (0)
>    - *A* man/as_labeller.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-236> (0)
>    - *A* man/autoplot.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-237> (0)
>    - *A* man/benchplot.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-238> (0)
>    - *A* man/borders.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-239> (0)
>    - *A* man/calc_element.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-240> (0)
>    - *M* man/checkAnimationTimeVar.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-241> (0)
>    - *M* man/checkExtraParams.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-242> (0)
>    - *M* man/checkForSSandCSasAesthetics.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-243> (0)
>    - *M* man/checkPlotForAnimintExtensions.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-244> (0)
>    - *M* man/checkPlotList.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-245> (0)
>    - *M* man/checkSingleShowSelectedValue.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-246> (0)
>    - *M* man/colsNotToCopy.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-247> (0)
>    - *A* man/continuous_scale.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-248> (0)
>    - *A* man/coord_cartesian.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-249> (0)
>    - *A* man/coord_fixed.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-250> (0)
>    - *A* man/coord_flip.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-251> (0)
>    - *A* man/coord_map.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-252> (0)
>    - *A* man/coord_munch.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-253> (0)
>    - *A* man/coord_polar.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-254> (0)
>    - *A* man/coord_trans.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-255> (0)
>    - *A* man/cut_interval.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-256> (0)
>    - *A* man/diamonds.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-257> (0)
>    - *A* man/discrete_scale.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-258> (0)
>    - *A* man/draw_key.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-259> (0)
>    - *A* man/economics.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-260> (0)
>    - *A* man/element_blank.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-261> (0)
>    - *A* man/element_grob.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-262> (0)
>    - *A* man/element_line.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-263> (0)
>    - *A* man/element_rect.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-264> (0)
>    - *A* man/element_text.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-265> (0)
>    - *A* man/expand_limits.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-266> (0)
>    - *A* man/facet.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-267> (0)
>    - *A* man/facet_grid.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-268> (0)
>    - *A* man/facet_null.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-269> (0)
>    - *A* man/facet_wrap.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-270> (0)
>    - *A* man/faithfuld.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-271> (0)
>    - *A* man/format.ggproto.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-272> (0)
>    - *A* man/fortify-multcomp.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-273> (0)
>    - *A* man/fortify.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-274> (0)
>    - *A* man/fortify.lm.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-275> (0)
>    - *A* man/fortify.map.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-276> (0)
>    - *A* man/fortify.sp.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-277> (0)
>    - *A* man/geom_abline.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-278> (0)
>    - *A* man/geom_bar.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-279> (0)
>    - *A* man/geom_bin2d.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-280> (0)
>    - *A* man/geom_blank.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-281> (0)
>    - *A* man/geom_boxplot.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-282> (0)
>    - *A* man/geom_contour.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-283> (0)
>    - *A* man/geom_count.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-284> (0)
>    - *A* man/geom_density.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-285> (0)
>    - *A* man/geom_density_2d.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-286> (0)
>    - *A* man/geom_dotplot.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-287> (0)
>    - *A* man/geom_errorbarh.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-288> (0)
>    - *A* man/geom_hex.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-289> (0)
>    - *A* man/geom_histogram.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-290> (0)
>    - *A* man/geom_jitter.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-291> (0)
>    - *A* man/geom_linerange.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-292> (0)
>    - *A* man/geom_map.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-293> (0)
>    - *A* man/geom_path.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-294> (0)
>    - *A* man/geom_point.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-295> (0)
>    - *A* man/geom_polygon.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-296> (0)
>    - *A* man/geom_quantile.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-297> (0)
>    - *A* man/geom_ribbon.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-298> (0)
>    - *A* man/geom_rug.Rd
>    <https://github.com/tdhock/animint2/pull/23/files#diff-299> (0)
>
> Patch Links:
>
>    - https://github.com/tdhock/animint2/pull/23.patch
>    - https://github.com/tdhock/animint2/pull/23.diff
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tdhock/animint2/pull/23>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA478qad3H7LtJJtgx-JU8VBXO2ys356ks5t81NJgaJpZM4UpEha>
> .
>

vivekktiwari commented:
@faizan-khan-iit Summary of changes made here: 

1. I have just merged the `ggplot2Animint` code from `validate-params` branch of yours, removed references to `ggplot2Animint`. 
2. Exported all `Description` and `Namespace` from `ggplot2Animint`, thanks for sharing your PRs that helped alot. 
2. I have also added `data` and `data-raw` files from `ggplot2Animint` code. 
3. There are major corrections in Examples and Documentation: `ggplot2-ggproto.Rd` is replaced with `animint2-ggproto.Rd`, also added all the things from there.

tdhock commented:
looks good in general, glad to see all tests are passing

in this branch ggplot2/ggplot2Animint is no longer required, right? however on the README under Installation it still says to install ggplot2 -- can you please remove that line from the README? and maybe add a sentence or two that explains that ggplot2 functions have been forked and are included

vivekktiwari commented:
@tdhock removed the line, but I don't know why Travis is failing now. seems like error from Travis side.

tdhock commented:
hi @vivekktiwari sorry it has taken me so long to look into this.

there was some problem with the test definition -- I had to add scale_y(breaks=something) in order to get the right number of grid lines to show up in the data viz -- they are now passing on my machine.

I also bumped the version and it looks like this is ready to merge with master right? (I am a bit confused about all of the other PRs you opened -- I assume this is the most relevant one -- please close the ones that are no longer relevant)

vivekktiwari commented:
hi @tdhock, sorry I have been inactive, all due to extra coursework, exams and placement, 

Yes! This is the most relevant one. #25 is work-in-progress, I will resume working in a month, I will soon close irrelevant PRs.

tdhock commented:
Hi vivek I pushed a new TEST_SUITE=CRAN which runs the CRAN checks.
hopefully we can get that working before merging with master.

On Thu, Oct 4, 2018 at 12:04 PM Vivek Kumar <notifications@github.com>
wrote:

> hi @tdhock <https://github.com/tdhock>, sorry I have been inactive, all
> due to extra coursework, exams and placement,
>
> Yes! This is the most relevant one. #25
> <https://github.com/tdhock/animint2/pull/25> is work-in-progress, I will
> resume working in a month, I will soon close irrelevant PRs.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tdhock/animint2/pull/23#issuecomment-427133990>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AA478uTx1H5uz4SUd1tMvpjtD-8UFyJVks5uhlu2gaJpZM4UpEha>
> .
>

vivekktiwari commented:
Is it feasible to separate the TEST_SUIT=CRAN from other test suits, as we won't be running major of the other test cases while submitting to CRAN, right?

tdhock commented:
right, for CRAN I execute
https://github.com/tdhock/animint2/blob/merge/build.sh which only runs the
compiler tests

On Fri, Oct 5, 2018 at 10:10 AM Vivek Kumar <notifications@github.com>
wrote:

> Is it feasible to separate the TEST_SUIT=CRAN from other test suits, as we
> won't be running major of the other test cases while submitting to CRAN,
> right?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tdhock/animint2/pull/23#issuecomment-427435482>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AA478iONH1GxJfbVZPvwtd7U1M_hHvFTks5uh5KCgaJpZM4UpEha>
> .
>

tdhock commented:
hey @vivekktiwari tests are now passing

can you please review the code and tell me if you think we should merge with master as it is right now, or do you think we need to add something else?

vivekktiwari commented:
@tdhock according to me, I  think it is ready to merge.

tdhock commented on README.org:
```diff
@@ -18,6 +18,9 @@ new syntax.
 ** Differences with old animint
 
+
+06-10-2018: =train_layout= [from ggplot2 - R/panel.R] function is now =g_train_layout=. =train_layout= [from animint2- R/z_facet.R] function is the same. Both are =internal functions=.
```
hey @vivekktiwari I moved this to NEWS since README should only contain info which is important for users (not info about internal functions)

tdhock commented:
merged! thanks so much for all your hard work @vivekktiwari
